course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -156,7 +156,9 @@
                             $scope.searchInProgress = true;
                         });
                         $scope.enableColumnSorting(false);
-                        var queryParameters = {};
+                        //filter the sites flagged to be excluded(get only ones with exclude_from_isites set to 0)
```

Comment:
(if possible, let's try to stick to 80-char-width lines, as we've discussed before)